cts/aggregation/operators/abs.json

```diff
@@ -0,0 +1,194 @@
+{
+  "simple": {
+    "request": {
+      "aggregate": "values",
+      "pipeline": [
+        {
+          "$match": {
+            "v": {
+              "$type": [
+                "int",
+                "long",
+                "double",
+                "decimal"
+              ],
+              "$gt": {
+                "$numberLong": "-9223372036854775808"
```

Comment:
`$match` is already using `$type` filter, why do you need this `$gt` condition? Can we remove it? `$abs` is able to handle numbers, null and missing field.

Reply:
The absolute value of int64-min (`-9223372036854775808`), overflows int64, which results in error.

Reply:
I suggest matching specific `_id` that case. Because float64 could have a smaller valid number right?

Reply:
@noisersup this comment wasn't addressed :wave: